exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -502,6 +502,8 @@ drill.exec.options: {
     store.format: "parquet",
     store.hive.optimize_scan_with_native_readers: false,
     store.json.all_text_mode: false,
+    store.json.writer.non_numeric_numbers: false,
+    store.json.reader.non_numeric_numbers: false,
```

Comment:
Any reason these are not enabled by default?

Reply:
I think we should stick to json standard and leave them switched off by default. If user get an exception we may show the option name which he want to switch on.

Reply:
As noted in DRILL-5949, options like this should be part of the plugin config (as in CSV), not session options. For now, let's leave them as session options and I'll migrate them to plugin config options along with the others.

Reply:
See discussion below. If they are off, then queries that use NaN and Infinity will fail until the user turns them on. Queries that don't use NaN and Infinity won't care. So, what is the advantage to failing queries unnecessarily?